api.bs

```diff
@@ -883,13 +895,17 @@ TODO
 
 ## Conversion Report Encryption ## {#encryption}
 
-TODO
+In addition to encrypting the histogram contribution, some data is bound to the report as authenticated data.  The authenticated data includes
+1.  The `requested_epsilon` and the `query_global_sensitivity` and the `p-norm` used to enforced the sensitivity.
```

Comment:
```suggestion
1.  The `requested_epsilon` and the `query_global_sensitivity` and the `p-norm` used to compute the sensitivity.
```

Do we really need an arbitrary p-norm, or is it enough to use a straightforward L1 norm?

Reply:
Yes, we can start with just the L1 norm for simplicity.

Reply:
will update here and in #28